BDArmory/Ammo/BulletInfo.cs

```diff
@@ -272,6 +283,55 @@ private static void PostProcessOldSubmunitionConfigs()
             }
             oldSubmunitionConfigs.Clear();
         }
+
+        private void ParseTypes()
+        {
+            if (tntMass > 0)
+                eHEType = explosive.ToLower() switch
+                {
+                    "standard" or "true" => PooledBulletTypes.Explosive,
+                    "shaped" => PooledBulletTypes.Shaped,
+                    _ => PooledBulletTypes.Slug
+                };
+            else
+                eHEType = PooledBulletTypes.Slug;
+
+            bulletDragType = bulletDragTypeName.ToLower() switch
+            {
+                "none" => BulletDragTypes.None,
+                "numericalintegration" => BulletDragTypes.NumericalIntegration,
+                "analyticestimate" => BulletDragTypes.AnalyticEstimate,
+                _ => BulletDragTypes.AnalyticEstimate
+            };
+
+            if (tntMass > 0 || beehive)
+            {
+                eFuzeType = fuzeType.ToLower() switch
+                {
+                    //Anti-Air fuzes
+                    "timed" => BulletFuzeTypes.Timed,
+                    "proximity" => BulletFuzeTypes.Proximity,
+                    "flak" => BulletFuzeTypes.Flak,
+                    //Anti-Armor fuzes
+                    "delay" => BulletFuzeTypes.Delay,
+                    "penetrating" => BulletFuzeTypes.Penetrating,
+                    "impact" => BulletFuzeTypes.Impact,
+                    "none" => BulletFuzeTypes.Impact,
+                    _ => BulletFuzeTypes.None
```

Comment:
``Impact`` or ``None`` aren't viable fuze types for beehive rounds (no point for the first, no function from the latter). Should probably default to ``Proximity``, if we want to be kind, ``Timed`` if not.

Reply:
Looks like the `"none" => BulletFuzeTypes.Impact` slipped past me, that should've been corrected in other cases to `BulletFuzeTypes.None`. No clue why anyone would want that but I feel like if someone actually writes "none" it should do what it says on the tin. Agreed for the default though, I was thinking of having
`_ => beehive ? BulletFuzeTypes.Timed: BulletFuzeTypes.Impact`
there.

Reply:
Hmm now that I think about it "none" might've been used for impact at some point, so I'll just add the conditional statement there too

Reply:
None = Impact was a backwards compatibility legacy thing for when fuze types got first implemented, yes. It's been long enough we could probably change it.